reliability-v2/tasks/Tasks.py

```diff
@@ -443,4 +445,42 @@ def check_netobserv_pods(self, user):
             else:
                 self.logger.error(f"Pods status fetch error: result {result}, rc - {rc}")
                 rc_return = 1
-        return (result, rc_return)
\ No newline at end of file
+        self.__log_result(rc)
+        return (result, rc_return)
+
+    # Get all pods in a namespace
+    def get_all_pods_in_namespace(self, user, namespace):
+        pod_names = {}
+        self.logger.info(f"[Task] User {user}: get all pods in a namespace")
+        # Get all pods in a particular namespace
+        (result, rc) = oc(
+            "get pods -n {namespace} -o jsonpath='{.items[*].metadata.name}'",
+            self.__get_kubeconfig(user),
+            ignore_log=True,
+            ignore_slack=True,
+        )
+        if rc == 0 and result != "":
+            self.logger.info(f"Pods in ns {namespace} are: {result}")  
+            pod_names = string.Split(string.Trim(result, "'"), " ")
+        return pod_names
```

Comment:
Let's return the rc as well `return (pod_names, rc)`. I hope this function used directly as a task.